tests/apis/platform/api.test.ts

```diff
@@ -764,4 +764,25 @@ describe("Platform", () => {
         expect(response).toBe(txID);
     });
 
+    test("getSubnets 1", async ()=>{
+        let resp: object[] = [{
```

Comment:
Should this be

```js
let resp: Array<object>
```

Unless this syntax is shorthand I haven't seen yet.

Reply:
This is a shorthand: https://www.typescriptlang.org/docs/handbook/basic-types.html#array

I prefer generics as its consistent across the entire codebase. Let's stick to that style.